src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
+  --header-height: 60px;
+  --hover-color: #00ACDC;
+
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  width: 100%;
+}
+
+.container {
+  margin: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  text-decoration: none;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: start;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  display: flex;
+  align-content: center;
+  align-items: center;
+  height: var(--header-height);
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)